src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -638,8 +663,14 @@ class ContentSidebar extends PureComponent<Props, State> {
         this.api.getCommentsAPI(false).createComment({
```

Comment:
All these API functions should ideally not be {} based, args should be concrete/specific.